.github/workflows/test_linux_jax_wheels.yml

```
@@ -0,0 +1,169 @@
+    steps:
+      - name: Checkout TheRock
+        uses: actions/checkout@08c6903cd8c0fde910a37f88322edcfb5dd907a8 # v5.0.0
+        with:
+          repository: ${{ inputs.repository || github.repository }}
+          ref: ${{ inputs.ref || '' }}
+
+      - name: Checkout rocm-jax (plugin + build scripts)
+        uses: actions/checkout@08c6903cd8c0fde910a37f88322edcfb5dd907a8 # v5.0.0
+        with:
+          path: jax
+          repository: rocm/rocm-jax
+          ref: ${{ inputs.jax_ref }}
+
+      - name: Setup Python
+        uses: actions/setup-python@e797f83bcb11b83ae66e0230d6156d7c80228e7c # v6.0.0
+        with:
+          python-version: "3.12"
+
+      - name: Prepare wheelhouse and download JAX wheels
+        shell: bash
+        working-directory: jax
+        run: |
+          python3 ../build_tools/fetch_wheels.py \
+            --cloudfront_url ${{ inputs.package_index_url }} \
+            --amdgpu_family ${{ inputs.amdgpu_family }} \
+            --dir "wheelhouse" \
+            --list_whls '${{ inputs.jax_whl_list }}'
+
+      - name: Compute ROCM_VERSION_SHORT
+        working-directory: jax
+        env:
+            ROCM_VERSION: ${{ inputs.rocm_version }}
+        run: |
+            # Extract major.minor.patch from ROCM_VERSION
+            ROCM_VERSION_SHORT="$(echo "${ROCM_VERSION}" | sed -E 's/^([0-9]+)\.([0-9]+)\.([0-9]+).*/\1.\2.\3/')"
+            echo "ROCM_VERSION_SHORT=${ROCM_VERSION_SHORT}" >> "$GITHUB_ENV"
+            echo "Full ROCm version: ${ROCM_VERSION}"
+            echo "Using semantic ROCm version: ${ROCM_VERSION_SHORT}"
+
+      - name: Build JAX Docker image
+        working-directory: jax
+        env:
+          ROCM_VERSION: ${{ inputs.rocm_version }}
+        run: |
+            python3 build/ci_build \
+            --rocm-version="${ROCM_VERSION_SHORT}" \
+            --therock-path="${{ inputs.tar_url }}" \
+            build_dockers \
+            -f ubu24
+            # Assign JAX_IMAGE to the expected image name produced by the build script
+            JAX_IMAGE="jax-ubu24.rocm$(echo "${ROCM_VERSION_SHORT}" | tr -d '.'):latest"
+            echo "JAX_IMAGE=${JAX_IMAGE}" >> "$GITHUB_ENV"
+            echo "Built JAX Docker image: ${JAX_IMAGE}"
+
+      - name: Checkout JAX tests repo (for extended tests)
+        uses: actions/checkout@08c6903cd8c0fde910a37f88322edcfb5dd907a8 # v5.0.0
+        with:
+          repository: rocm/jax
+          ref: ${{ inputs.jax_ref }}
+          path: jax/jax_tests
+
+      - name: Run unit tests
+        working-directory: jax
+        env:
+          ROCM_VERSION: ${{ inputs.rocm_version }}
+        run: |
+            # Run the tests and capture output
+            CORETEST_OUTPUT=$(python3 build/ci_build test \
+            "${JAX_IMAGE}" \
+            --test-cmd "pytest jax_tests/tests/core_test.py" 2>&1)
+            echo "$CORETEST_OUTPUT" > coretest_output.txt
+            cat coretest_output.txt >> $GITHUB_STEP_SUMMARY
```

This step doesn't make sense to me. Why capture the output just to dump it out to `GITHUB_STEP_SUMMARY`?

How about this, streaming output to stdout as is the default?

```suggestion
      - name: Run unit tests
        working-directory: jax
        env:
          ROCM_VERSION: ${{ inputs.rocm_version }}
        run: |
            python3 build/ci_build test \
              "${JAX_IMAGE}" \
              --test-cmd "pytest jax_tests/tests/core_test.py"
```

You could _also_ output to a file and then upload that file, as we do with https://github.com/ROCm/TheRock/blob/main/build_tools/teatime.py and https://github.com/ROCm/TheRock/blob/main/build_tools/github_actions/post_build_upload.py for our CMake builds.

Only capturing the output in a file and then posting makes the logs on the test step itself completely empty: https://github.com/ROCm/TheRock/actions/runs/19225042724/job/54954865305#step:9:1. For long running test processes being able to see intermediate output as it appears is very useful, and we shouldn't have just one workflow (test_linux_jax_wheels.yml) handling logging differently.